### ISSUE TITLE
Uncaught ImportError: cannot import name 'processing'


### ISSUE THREAD

**Issue by RobyHope** [NationalSecurityAgency/qgis-kmltools-plugin#11]
Impossibile caricare il plugin 'kmltools' a causa di un errore chiamando il metodo classFactory()
Steps to reproduce
 1. First 
 2. Second 
 3. Third 

```
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 335, in startPlugin
    plugins[packageName] = package.classFactory(iface)
  File "/home/osgeolive/.local/share/QGIS/QGIS3/profiles/default/python/plugins/kmltools/__init__.py", line 9, in classFactory
    from .kmltools import KMLTools
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 685, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/osgeolive/.local/share/QGIS/QGIS3/profiles/default/python/plugins/kmltools/kmltools.py", line 28, in <module>
    from .provider import KmlToolsProvider
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 685, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/osgeolive/.local/share/QGIS/QGIS3/profiles/default/python/plugins/kmltools/provider.py", line 7, in <module>
    from .convertGroundOverlays import ConvertGroundOverlayAlgorithm
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 685, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/osgeolive/.local/share/QGIS/QGIS3/profiles/default/python/plugins/kmltools/convertGroundOverlays.py", line 17, in <module>
    from qgis import processing
ImportError: cannot import name 'processing'

```
kmltools 3.1.7, QGIS 3.4.10-Madeira on Linux 5.0.0-23-generic

**Comment by hamiltoncj** [comment]
Thank you for letting me know. That was caused by QGIS API changes from 3.4 to where we are now, but in the process of testing some of the advanced GDAL features used in these new tools, I found they are not available until QGIS 3.14. Currently I have deleted KML Tools 3.1.7 and replaced it with 3.1.8, but it requires QGIS 3.14 or later. I would suggest reverting back to KML Tools 3.1.6 or upgrading your QGIS to 3.16.4. Next week I will investigate to see if I can back port these capabilities for older versions of QGIS or come up with some other solution for this.
